<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectInteractionContainer</class>
 <widget class="QWidget" name="ObjectInteractionContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>2400</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_w1" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <widget class="QWidget" name="widget_w1b" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_w2" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <widget class="QWidget" name="widget_w2b" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_w3" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <widget class="QWidget" name="widget_w3b" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_w4" native="true">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>300</height>
      </size>
     </property>
     <widget class="QWidget" name="widget_w4b" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
